File: microservices.springframework/chatbot-service/src/main/java/swd392/chatbotservice/application/exception/ValidationErrorResponse.java
package swd392.chatbotservice.application.exception;

import lombok.Builder;
import lombok.Data;
import java.time.LocalDateTime;
import java.util.Map;

@Data
@Builder
public class ValidationErrorResponse {
    private int status;
    private String message;
    private Map<String, String> errors;
    private LocalDateTime timestamp;
}

File: microservices.springframework/identity-service/src/main/kotlin/swd392/identityservice/application/exception/ValidationErrorResponse.kt
package swd392.identityservice.application.exception

import lombok.Builder
import lombok.Data
import java.time.LocalDateTime

@Data
@Builder
class ValidationErrorResponse (
    val status: Int = 0,
    val message: String? = null,
    val errors: MutableMap<String?, String?>? = null,
    val timestamp: LocalDateTime? = null
)